lib/sacastats_web/templates/character/weapons.html.heex

```diff
@@ -556,6 +556,22 @@
     </tbody>
 </table>
 
+
+
+<link href="https://unpkg.com/bootstrap-table@1.21.0/dist/extensions/sticky-header/bootstrap-table-sticky-header.css" rel="stylesheet">
```

Comment:
This block of script imports will need to be added at the bottom of any template that implements out `flex-bootstrap-table`. Is there any way to make this modular enough to be able to be obtained by one call accessible by all templates?

Reply:
Hmm maybe you can include that as part of all of our other main CSS links?

Reply:
The problem with that is there are also a bunch of script imports as well. Also trying to avoid importing things we don't need to speed up loading as tables are not on every page and they are pretty big files.

Reply:
Ahh okay that makes sense. Yeah I'm not sure if there's a better way to selectively import unfortunately - maybe some documentation somewhere in the table would be helpful?

Reply:
Well, we can selectively import. This is by only adding it to the templates that need them like I've been doing with the current scripts and stylesheets. I was just wondering if we could make a method like this:
```elixir
def import_flex_bootstrap_table(_assigns) do
    ~H"""
    <link rel="stylesheet" href="https://unpkg.com/@fortawesome/fontawesome-free@5.12.1/css/all.min.css">
    <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/3.2.1/css/font-awesome.min.css" rel="stylesheet" />
    <link href="https://cdn.jsdelivr.net/gh/akottr/dragtable@master/dragtable.css" rel="stylesheet">
    <link href="https://unpkg.com/bootstrap-table@1.21.0/dist/bootstrap-table.min.css" rel="stylesheet">

    <script src="https://cdn.jsdelivr.net/npm/jqueryui@1.11.1/jquery-ui.js"></script>
    <script src="https://cdn.jsdelivr.net/gh/akottr/dragtable@master/jquery.dragtable.js"></script>
    <script src="https://unpkg.com/bootstrap-table@1.21.0/dist/bootstrap-table.min.js"></script>
    <script src="https://unpkg.com/bootstrap-table@1.21.0/dist/extensions/reorder-columns/bootstrap-table-reorder-columns.min.js"></script>
    <script src="https://unpkg.com/bootstrap-table@1.21.0/dist/extensions/sticky-header/bootstrap-table-sticky-header.min.js"></script>

    <script type="module" src={Routes.static_path(@conn, "/assets/dist/flex-bootstrap-table/flex-bootstrap-table.js")}> </script>
    """
  end
```
  and just call it at the end of the template like this when there is a table in that specific template:
```elixir
<%=SacaStats.Utils.import_flex_bootstrap_table(assigns)%>
```
I just wasn't sure where to put that method. Maybe `Utils`, but wasn't sure.

Reply:
Ohh I see, I like that idea. I think you could put that function in `CharacterView` - or if you think you'll use the table in non-character pages as well, `Utils` is probably a good choice too